10.IDR_metadata/scripts/0.preprocess_profiles.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+
+
+# In[2]:
+
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+import papermill as pm
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+# In[3]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+
+
+# In[4]:
+
+
+# Define inputs
+feature_file = pathlib.Path(f"../../data/{cell_type}_sc_norm.parquet")
+feature_df = pd.read_parquet(feature_file)
+
+
+# In[ ]:
```

Comment:
It looks like the previous cell included a keyboard interrupt. Is this expected?

The first cell of this notebook (markdown documentation) would be the place to describe if it is expected. I suspect that because you're using `papermill` we shouldn't see any cells executed.

This documentation would be super helpful to be included in the first cell and also the module-specific README.

Reply:
Addressed